package.json

```diff
@@ -41,7 +41,7 @@
     "codeclimate-test-reporter": "^0.4.0",
     "colors": "^1.1.2",
     "css-loader": "^0.26.1",
-    "d2": "^26.1.7",
+    "d2": "^26.1.12",
```

Comment:
I meant that packages.json/yarn.lock should be equal in d2 and d2-ui. Indeed we can set directly the latest in package.json